packages/config/src/index.ts

```diff
@@ -51,6 +51,7 @@ export const types = Object.assign({
   loglevel: ['silent', 'error', 'warn', 'info', 'debug'],
   'modules-dir': String,
   'network-concurrency': Number,
+  'node-linker': ['pnp'],
```

Comment:
`node‑modules` should also be a valid value:
```suggestion
  'node-linker': ['node-modules', 'pnp'],
```

Reply:
I don't think that is a good idea currently because Yarn 2 is adding a node-linker that will create a pnpm-like node_modules: https://github.com/yarnpkg/berry/issues/1845

I don't know how it will be named but that is what we'll need to use as the default value of node-linker. `node-linker=node-modules` is a traditional node_modules that is created by npm and Yarn 1. pnpm does not support that. I think we can even throw an error on that option.